### DISCUSSION TITLE
issue: I can't render the input that is inside the controller using Jest and React Native Testing Library

### DISCUSSION THREAD

**Discussion by ildaneta** [react-hook-form/react-hook-form#9237] [Q&A]
### Version Number

^7.37.0

### Codesandbox/Expo snack

https://codesandbox.io/s/react-native-input-test-hvzuqm

### Steps to reproduce

I can't render the input that is inside the Controller, on Input component.

### Expected behaviour

When I debug the test, it only shows me the label text, but the input that is inside the Controller, is not rendered inside the test.

![Captura de Tela 2022-10-17 às 19 11 45](https://user-images.githubusercontent.com/21963291/196240627-1f8ef482-a640-466c-afa7-a5f613d5c7a4.png)

But, when I comment the controller that are around the input, and debug again the test, the input is rendered:
![Captura de Tela 2022-10-17 às 19 14 39](https://user-images.githubusercontent.com/21963291/196241194-b22b43cb-3344-4d77-82b8-7b70a627ec5c.png)

![Captura de Tela 2022-10-17 às 19 14 07](https://user-images.githubusercontent.com/21963291/196241076-188a8d02-8807-4150-902e-896b9bd6e561.png)

My test is missing something?
Could you help me please to check my input rendered?

Regards,

#### Package.json

```ts
{
  "name": "",
  "version": "0.3.0",
  "main": "index.js",
  "scripts": {
    "start": "expo start --dev-client",
    "android": "expo run:android",
    "ios": "expo run:ios",
    "web": "expo start --web",
    "test": "jest",
  },
  "dependencies": {
    "@hookform/resolvers": "^2.9.9",
    "@react-native-async-storage/async-storage": "^1.17.10",
    "@react-native-community/netinfo": "^9.3.4",
    "@react-navigation/native": "^6.0.13",
    "@react-navigation/native-stack": "^6.9.0",
    "@types/react": "~18.0.21",
    "@types/react-native": "~0.69.1",
    "babel-plugin-inline-dotenv": "^1.7.0",
    "expo": "~46.0.16",
    "expo-splash-screen": "~0.16.2",
    "expo-status-bar": "~1.4.0",
    "jest": "^29.1.2",
    "jest-expo": "^46.0.1",
    "react": "18.0.0",
    "react-dom": "18.0.0",
    "react-hook-form": "^7.37.0",
    "react-native": "0.69.6",
    "react-native-safe-area-context": "4.3.1",
    "react-native-screens": "~3.15.0",
    "react-native-web": "~0.18.7",
    "styled-components": "^5.3.6",
    "typescript": "^4.6.3",
    "yup": "^0.32.11"
  },
  "devDependencies": {
    "@babel/core": "^7.12.9",
    "@testing-library/jest-native": "^5.0.0",
    "@testing-library/react-native": "^11.2.0",
    "@types/jest": "^29.1.2",
    "@types/styled-components-react-native": "^5.1.3",
    "react-test-renderer": "^18.2.0"
  },
  "jest": {
    "preset": "jest-expo",
    "setupFilesAfterEnv": [
      "@testing-library/jest-native/extend-expect"
    ],
    "collectCoverage": true,
    "collectCoverageFrom": [
      "**/*.{ts,tsx}",
      "!**/coverage/**",
      "!**/node_modules/**",
      "!**/babel.config.js",
      "!src/**/*styles.ts",
      "!src/routes/*",
      "!<rootDir>/App.tsx",
      "!src/components/Divider/index.tsx"
    ]
  },
  "private": true
}

```
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Moshyfawn** [comment]
I see `Controller: () => <></>` code. Is it intended?

**Comment by bluebill1049** [comment]
refer to the examples here: https://github.com/react-hook-form/react-hook-form/blob/master/src/__tests__/controller.test.tsx

> I see Controller: () => <></> code. Is it intended?

That pretty much prevents `Controller` component from doing anything.

**Comment by MariusGG** [comment]
Hi @ildaneta, I was wondering if you ever managed to resolve this issue, as I'm currently dealing with the same problem.
When running my test, I get no value for the rendered output of the `Controller` 
Would very much appreciate any suggestions or use links.

**Comment by sison-bic** [comment]
Hi, I have the same issue, how can i fix this?
